src/components/calculator.js

```diff
@@ -0,0 +1,145 @@
+                  +/-
+                </button>
+              </td>
+
+              <td>
+                <button type="button" className="non-operators">
+                  %
+                </button>
+              </td>
+
+              <td>
+                <button type="button" className="operators">
+                  ÷
+                </button>
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <button type="button" className="non-operators">
+                  7
+                </button>
+              </td>
+
+              <td>
+                <button type="button" className="non-operators">
+                  8
+                </button>
+              </td>
+
+              <td>
+                <button type="button" className="non-operators">
+                  9
+                </button>
+              </td>
+
+              <td>
+                <button type="button" className="operators">
+                  x
+                </button>
+              </td>
+            </tr>
+
+            <tr>
+              <td>
+                <button type="button" className="non-operators">
+                  4
+                </button>
+              </td>
+
+              <td>
+                <button type="button" className="non-operators">
+                  5
+                </button>
+              </td>
+              <td>
+                <button type="button" className="non-operators">
+                  6
+                </button>
+              </td>
+              <td>
+                <button type="button" className="operators">
+                  -
+                </button>
+              </td>
+            </tr>
+            <tr>
+              <td>
+                <button type="button" className="non-operators">
+                  1
+                </button>
+              </td>
+              <td>
+                <button type="button" className="non-operators">
+                  2
+                </button>
+              </td>
+              <td>
+                <button type="button" className="non-operators">
+                  3
+                </button>
+              </td>
+              <td>
+                <button type="button" className="operators">
+                  +
+                </button>
+              </td>
+            </tr>
+
+            <tr>
+              <td colSpan="2">
+                <button type="button" className="non-operators">
+                  0
+                </button>
+              </td>
+
+              <td>
+                <button type="button" className="non-operators">
+                  .
+                </button>
+              </td>
+
+              <td>
+                <button type="button" className="operators">
+                  =
+                </button>
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    );
+  }
+}
+
+export default Calculator;
```

Comment:
- [ ]  It would be better for your calculator.js to be a functional component, functional components are the modern ways to write components 👍🏿